Dart/src/com/jetbrains/lang/dart/analyzer/DartAnalysisServerService.java

```diff
@@ -403,6 +432,7 @@ public void run() {
         }
       }
     }
+    myFilePathsWithUnsentChanges.clear();;
```

Comment:
Wouldn't it be more correct to clear this set in updateFilesContent() or even in UpdateContentConsumer.onResponse() inside updateFilesContent()?